src/verinfast/config.py

```diff
@@ -222,7 +222,10 @@ def __init__(self, cfg_path: str = None) -> None:
         if (
             "repos" not in self.config and
             "local_repos" not in self.config and
-            "modules" not in self.config  # If cloud specified it's here.
+            (
+                "modules" not in self.config or
+                "cloud" not in self.config["modules"]
+            )
```

Comment:
Just curious what this is doing.

Reply:
Fixes unrelated existing bug that broke local path from command line

Reply:
To be fair, local path will still be broken with cloud in a config, but that was outside the scope of this. Needed to fix this since the unit test wasn't running.